packages/apollo/README.md

```diff
@@ -217,7 +218,7 @@ OPTIONS
 
   --header=header                        Additional header to send to server for introspectionQuery. May be used
                                          multiple times to add multiple headers. NOTE: The `--endpoint` flag is REQUIRED
-                                         if using the `--header` flag.
+                                         if using the `--header` flag. NOTE: `--header` flags must preceed `--endpoint`.
```

Comment:
I'm not certain this is true. How are you using the multiple header flags? If so, I imagine this would be a bug, not a documentation fix.